apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1,14 +1,21 @@
 {
   "500": "Internal Server Error",
+  "__agents__agents_and__count__conversations__period__": "{{agents}} agents and {{count}} conversations, {{period}}",
   "__count__empty_rooms_will_be_removed_automatically": "{{count}} empty rooms will be removed automatically.",
   "__count__empty_rooms_will_be_removed_automatically__rooms__": "{{count}} empty rooms will be removed automatically:<br/> {{rooms}}.",
   "__count__message_pruned": "{{count}} message pruned",
   "__count__message_pruned_plural": "{{count}} messages pruned",
+  "__count__conversations__period__": "{{count}} conversations, {{period}}",
+  "__count__tags__period__": "{{count}} tags, {{period}}",
```

Comment:
I believe this is not reflecting what it was intended for, are we trying to show the quantity of the tags or the total conversations that contain tags?